src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <div class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
+        <input
+          class="search-bar__search search-bar__search--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="search-bar search-bar__small"
+        data-qa="small"
+      >
```

Comment:
The same goes for the `search-bar__small` class. Consider changing this to `search-bar--small` or another descriptive class name.